metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -337,10 +339,12 @@ def metadata_read_capability_test(
         for project_id in project_ids:
             try:
                 logger.info((f"Metadata read capability test for project {project_id}"))
-                client: bigquery.Client = config.get_bigquery_client()
-                assert client
+                bq_client: bigquery.Client = config.get_bigquery_client()
+                assert bq_client
+                proj_client: resourcemanager_v3.Client = config.get_projects_client()
+                assert proj_client
```

Comment:
Please put this under a config disabled by default. We don't want folks ingestions to break due to new permissions required as mentioned in other comment.